lti_consumer/lti_xblock.py

```diff
@@ -645,6 +646,22 @@ def workbench_scenarios():
         ]
         return scenarios
 
+    @staticmethod
+    def _get_statici18n_js_url(loader): # pragma: no cover
```

Comment:
I ended up marking this as no-cover in tests. This just got really messing trying to gain coverage on for little value since some of these functions are mocked in a test environment.